web/styles/zulip.css

```diff
@@ -1321,7 +1321,7 @@ div.toggle_resolve_topic_spinner .loading_indicator_spinner {
 .empty-feed-notice-title {
     font-size: 1.5em;
     font-weight: 400;
-    line-height: 1;
+    line-height: 1.2;
```

Comment:
This was just recently set in #32123, so unless there's been a design conversation in CZO about changing this, I'd just leave it as is.